packages/components/tests/dummy/app/templates/components/page-header.hbs

```diff
@@ -0,0 +1,358 @@
+            <Hds::Breadcrumb::Item @text="Access control (IAM)" @icon="users" />
+          </Hds::Breadcrumb>
+        </PH.Breadcrumb>
+        <PH.IconTile @icon="users" @color="terraform" />
+        <PH.Actions>
+          <Hds::Dropdown as |DD|>
+            <DD.ToggleButton @text="Manage users" @color="secondary" />
+            <DD.Interactive @text="Assign roles" @icon="user" />
+            <DD.Interactive @text="Batch edit" @icon="edit" />
+            <DD.Interactive @text="Delete user" @icon="trash" />
+          </Hds::Dropdown>
+          <Hds::Button @text="Add user" @icon="plus" @iconPosition="leading" />
+        </PH.Actions>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+    </SF.Item>
+
+    <SF.Item @label="With custom metadata">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Custom metadata</PH.Title>
+        <PH.Subtitle>beta-feature-cluster</PH.Subtitle>
+        <PH.Generic>
+          <Hds::Alert @type="compact" @color="warning" as |A|>
+            <A.Title>Beta feature</A.Title>
+            <A.Description>
+              This feature is in beta and is subject to change. Refer to the
+              <Hds::Link::Inline @href="#" @icon="external-link">HCP documentation</Hds::Link::Inline>
+              for more information and roadmap.
+            </A.Description>
+          </Hds::Alert>
+        </PH.Generic>
+        <PH.Actions>
+          <Hds::Button @text="Get started" @icon="beaker" @iconPosition="leading" />
+        </PH.Actions>
+        <PH.Badges>
+          <Hds::Badge @text="Beta feature" @icon="wand" @color="highlight" />
+        </PH.Badges>
+      </Hds::PageHeader>
+
+    </SF.Item>
+
+  </Shw::Flex>
+
+  <Shw::Divider @level={{3}} />
+
+  <Shw::Text::H2>Overflow examples and edge cases</Shw::Text::H2>
+
+  <Shw::Text::Body @tag="p">
+    These examples are meant to showcase the flexiblity, responsiveness, and potential gotchas related to the layout and
+    fluidity of the component.
+  </Shw::Text::Body>
+
+  <Shw::Flex @direction="column" as |SF|>
+
+    <SF.Item @label="Long title">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Boundary cluster access control and clusters</PH.Title>
+        <PH.IconTile @icon="boundary-color" @color="boundary" />
+        <PH.Actions>
+          <Hds::Dropdown as |DD|>
+            <DD.ToggleButton @text="Manage users" @color="secondary" />
+            <DD.Interactive @text="Assign roles" @icon="user" />
+            <DD.Interactive @text="Batch edit" @icon="edit" />
+            <DD.Interactive @text="Delete user" @icon="trash" />
+          </Hds::Dropdown>
+          <Hds::Button @text="Add user" @icon="plus" @iconPosition="leading" />
+        </PH.Actions>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+    </SF.Item>
+
+    <SF.Item @label="Long title + badge">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Long page title with a badge ipsum dolor sit amet</PH.Title>
```

Comment:
We want to capture the wrapping when the page is snapshotted by Percy (to detect possible regressions in future) so the text needs to wrap at full-page width

```suggestion
        <PH.Title>Page title with a very very very very long string that can span multiple lines</PH.Title>
```